index.html

```diff
@@ -16,9 +16,50 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="background-color-head">
+    <div class="container font">
+      <div class="row">
+        <div class="column-full">
+          <h1>Code Journal</h1>
+        </div>
+      </div>
+    </div>
   </header>
-  <main>
+  <main class="container font">
+    <div class="row">
+      <div class="column-full">
+        <h1>New Entry</h1>
+      </div>
+    </div>
```

Comment:
You should move this inside the `data-view="entry-form"` element, it is part of the entry form view.